test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua

```diff
@@ -1646,4 +1730,25 @@ function m.reRegisterAppsCustom_AnotherRPC(pTimeToRegApp2, pRPC)
   end
 end
 
+--[[ @isSubscribed: send OnInteriorVehicleData
+--! @parameters:
+--! pAppId - application number (1, 2, etc.)
+--! pIsExpApp1 - true(default) - if it's expected notification on mobile app1
+--! pIsExpApp2 - true - if it's expected notification on mobile app2
+--! pModuleType - module type to subscribe for
+--! pModuleId - module id to subscribe for
+--! @return: none
+--]]
+function m.isSubscribed(pIsExpApp1, pIsExpApp2, pModuleType, pModuleId)
+  if pIsExpApp1 == nil then pIsExpApp1 = true end
+  pModuleType = pModuleType or moduleTypeValue
+  pModuleId = pModuleId or m.getModuleControlData(pModuleType,1).moduleId
+  local occurences2 = pIsExpApp2 == true and 1 or 0
+  rc.rc.isSubscribed(pModuleType, pModuleId, 1, pIsExpApp1)
+  if pIsExpApp2 ~= nil then
+    local params = m.getActualModuleIVData(pModuleType, pModuleId)
```

Comment:
@GetmanetsIrina Please move definition of `occurences2` into `if pIsExpApp2 ~= nil` block
```suggestion
  rc.rc.isSubscribed(pModuleType, pModuleId, 1, pIsExpApp1)
  if pIsExpApp2 ~= nil then
    local occurences2 = pIsExpApp2 == true and 1 or 0
    local params = m.getActualModuleIVData(pModuleType, pModuleId)
```